README.md

```diff
@@ -25,48 +25,49 @@
+1. Design your architecture using common patterns
 2. Define it into JSON config files
 3. Let Newque handle all your message needs
 4. Develop and iterate faster instead of reinventing the wheel
 
 Newque is configured with JSON files that are loaded on startup. The format of those JSON files is documented [below](https://github.com/newque/newque#configuration-files).
 
-Newque is built around the concept of Listeners and Channels (also known as "topics"). Each Channel can be accessed from zero or more Listeners. Listeners expose a protocol ([HTTP](https://github.com/newque/newque#http) or [ZMQ](https://github.com/newque/newque#http)) over a port. Both protocols have pros and cons. Each Channel has a Backend where messages are sent before they can be read. Messages are binary blobs; it is possible to send anything from UTF-8 text to images, JSON, compressed video, etc.
+Newque is built around the concept of Listeners and Channels (sometimes known as "topics"). Each Channel can be accessed from zero or more Listeners. Listeners expose a protocol ([HTTP](https://github.com/newque/newque#http) or [ZMQ](https://github.com/newque/newque#zmq)) over a port. Both protocols have pros and cons. Each Channel has a Backend where messages are sent before they can be read. Messages are binary blobs; it is possible to send anything from UTF-8 text to images, JSON, compressed video, etc.
 
 ### Key Benefits
 
 Using Newque can afford you:
 
-- **Peace of mind**: you no longer have to ask yourself worrisome questions such as: 'Will my data collector run if one of my agents accidentally disconnects?'
+- **Peace of mind**: you no longer have to ask yourself worrisome questions such as: 'What happens if one of my data processing servers goes down?'
 - **Fast architecture prototyping**: it takes seconds to change the entire flow of data in your system to try different architectures. Avoid having to reinvent the wheel, or writing complicated and error-prone code to move data around.
-- **Time**: queue management over failed instances, pending queues, etc require hours of development and testing, and thousands of lines of code. Newque handles it all.
+- **Time**: queue management over failed instances, pending queues, etc require hours of development and testing. Newque handles it all.
 - **Fast application performance**
 
 ### Common Use Case
 
-Imagine clients (producers) recording events in an application. In this scenario, events happen continuously and the producers stream those single messages to Newque on a Channel (let's call it "Main") using the local disk as its Backend. This log allows the user to replay events later.
+Imagine clients (producers) recording events in an application. In this scenario, events happen continuously and the producers stream those single messages to Newque on a Channel (let's call it "Main") using the local disk as its Backend. This log allows the user to replay events later by Reading messages from that Backend.
```

Comment:
On second thought, I'm guessing these are capitalized to denote an action within the Newque model.